src/assets/custom/css/_sass/_soft-mod-cheat-sheet-cta-banner.scss

```diff
@@ -0,0 +1,36 @@
+.soft-mod-cheat-sheet-cta-block-image-en {
+  @include small {
+    display: none;
+  }
+
+  @include medium-large-and-extra-large {
+    background-image: url("#{$site-base-url}/assets/custom/img/softmod/bg-cta-banneer-business-case-for-change-en.jpg");
+    background-position: top left;
+    background-size: cover;
+    float: left;
+    height: 400px;
+    width: 50%;
+  }
+}
+
+.soft-mod-cheat-sheet-cta-block-image-es {
+  @include small {
+    display: none;
+  }
+
+  @include medium-large-and-extra-large {
+    background-image: url("#{$site-base-url}/assets/custom/img/softmod/bg-cta-banneer-business-case-for-change-es.jpg");
+    background-position: top left;
+    background-size: cover;
+    float: left;
+    height: 400px;
+    width: 50%;
+  }
```

Comment:
The only thing that varies between the `en` and `es` versions is the image. The rest of the styles could be shared